src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const EVERY_DAY_COST = 40;
+  const DISCOUNT_AFTER_3_DAYS = 20;
+  const DISCOUNT_AFTER_7_DAYS = 50;
+
+  if (days <= 2) {
```

Comment:
The condition `days <= 2` can be misleading because it implies that there might be a different calculation for `days == 0` or `days < 0`. It would be safer to check for `days > 0` to ensure that negative values or zero do not produce an incorrect calculation.